hooks/useLessonNav.ts

```diff
@@ -0,0 +1,47 @@
+import { usePathname } from 'next/navigation'
+import { chapters } from 'content'
+
+const handlePath = (pathName: string) => {
```

Comment:
Rename this to `getPathData` or something similar, `handlePath` does not imply it returns some data about the path.